recipes-debian/sources/connman-ui.inc

```diff
@@ -0,0 +1,19 @@
+# This is generated by debian-source.bbclass
+DPV = "0~20150623-1"
+DPV_EPOCH = ""
+REPACK_PV = "0~20150623"
+PV = "0~20150623"
+
+DEBIAN_SRC_URI = " \
+    ${DEBIAN_MIRROR}/main/c/connman-ui/connman-ui_0~20150623-1.dsc;name=connman-ui_0_20150623-1.dsc \
```

Comment:
According to meta/recipes-connectivity/connman/connman-gnome_0.7.bb use git://github.com/connectivity/connman-gnome.git as upstream source repo. However, connman-ui is different project which homepage is https://github.com/tbursztyka/connman-ui. 

So, connman-gnome and connman-ui are different software,. Therefore you need to add connman-ui package instead of connman-gnome. Then change recipes which depends on connman-gnome if such files exist. 

Also this is most important point, please describe why choose connman-ui instead of connman-gnome(AFAIK debian buster doesn't provide connman-gnome package) in commit message.